### ISSUE TITLE
Unneeded action icon sizes in Numix

### ISSUE THREAD

**Issue by palob** [numixproject/numix-icon-theme#1428]
Comparing with other themes and looking at the icons in GUIs of many applications it appears we actually don't need action (maybe other contexts as well) icons in sizes ``48`` and ``64``.

Apart from the fact that at least the GNOME ecosystem has moved to symbolic icons there simply are no apps which use action icons that big. Calibre used to have that strange big icons (48px I believe), some other apps allow for bigger action icons via user settings.

For this case the 24px/32px icons can be scaled up, no need for duplicates on our side.

The bigger sizes aren't needed for HiDPI scaling either, that's what the ``@2x`` symlinks are for.

Are there any reasons not to simply remove (or replace by symlinks) the big icons? As far as I can see they create an unneeded maintenance burden and add unneeded complexity when adding new icons.

**Comment by palob** [comment]
(Papirus)[https://github.com/PapirusDevelopmentTeam/papirus-icon-theme] don't even have 32px action icons, they just scale up the 16px ones.

Same for *vinceliuice's* (vimix)[https://github.com/vinceliuice/vimix-icon-theme] theme and many of their other themes.

So I don't think there removing the ``48``and ``64`` action icons can do any harm.

If the ``32`` ones (which to some likeliness are still used) were to be removed, possibly ``index.theme`` and/or symlinks would need to be adjusted so the scaled up ``16`` ones are used instead.

**Comment by palob** [comment]
I'm pulling the trigger here, fingers crossed. Removed ``48/actions``, ``64/actions`` and ``64/status`` as well as the symlinked sizes above 64 (96,128).
Not entirely sure when to use ``Fixed`` and when ``Scaled`` in ``index.theme``, ``Scaled`` may cause icons to be used in wrong places, e.g. a 64 px icon may be used in a 144px big spot (if existing) where the 48px one is suited better.

Maybe we need ``@3x`` or even ``@x4x`` HiDPI scaling symlinks but last time I checked with 300% display scaling (normally I use 200%) I did not look like we would need those.
